<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpWebBrowserPage</class>
 <widget class="QWidget" name="HelpWebBrowserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>904</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure web browser&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line17">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="singleHelpWindowCheckBox">
     <property name="toolTip">
      <string>Select to use a single help browser window only</string>
     </property>
     <property name="text">
      <string>Use single web browser window</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="saveGeometryCheckBox">
     <property name="toolTip">
      <string>Select to save the window size and position</string>
     </property>
     <property name="text">
      <string>Save size and position upon exit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="webSuggestionsCheckBox">
     <property name="toolTip">
      <string>Select to enable suggestions for web searches</string>
     </property>
     <property name="text">
      <string>Show suggestions for web searches</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="startupGroupBox">
     <property name="title">
      <string>Startup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>On startup:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QComboBox" name="startupCombo">
        <property name="toolTip">
         <string>Select the startup behavior</string>
        </property>
        <item>
         <property name="text">
          <string>Show home page</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Show empty page</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Home Page:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="homePageEdit">
        <property name="toolTip">
         <string>Enter the desired home page</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="setCurrentPageButton">
        <property name="toolTip">
         <string>Press to set the current page as the home page</string>
        </property>
        <property name="text">
         <string>Set to current page</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="defaultHomeButton">
        <property name="toolTip">
         <string>Press to set the default home page</string>
        </property>
        <property name="text">
         <string>Set to default home page</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>160</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Scheme</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Default Scheme:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="defaultSchemeCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the default scheme</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Default Scheme&lt;/b&gt;&lt;p&gt;Select the default scheme. This scheme is prepended to URLs, that don't contain one.&lt;/p&gt;</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="privacyGroup">
     <property name="title">
      <string>Privacy</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="javaCheckBox">
        <property name="toolTip">
         <string>Select to enable Java</string>
        </property>
        <property name="text">
         <string>Enable Java</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="javaScriptCheckBox">
        <property name="toolTip">
         <string>Select to enable JavaScript</string>
        </property>
        <property name="text">
         <string>Enable JavaScript</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="jsOpenWindowsCheckBox">
        <property name="toolTip">
         <string>Select to allow JavaScript to open windows</string>
        </property>
        <property name="text">
         <string>JavaScript can open windows</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="jsClipboardCheckBox">
        <property name="toolTip">
         <string>Select to allow JavaScript to access the clipboard</string>
        </property>
        <property name="text">
         <string>JavaScript can access clipboard</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="pluginsCheckBox">
        <property name="toolTip">
         <string>Select to enable plugins in web pages</string>
        </property>
        <property name="text">
         <string>Enable Plugins</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Security</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="savePasswordsCheckBox">
        <property name="toolTip">
         <string>Select to save passwords</string>
        </property>
        <property name="text">
         <string>Save passwords</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dnsPrefetchCheckBox">
        <property name="toolTip">
         <string>Select to enable DNS prefetch</string>
        </property>
        <property name="text">
         <string>Use DNS prefetching to improve page loading</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>History</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Remove history items:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="expireHistory">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the period for expiration of history entries</string>
        </property>
        <item>
         <property name="text">
          <string>After one day</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After one week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After two weeks</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After one month</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>After one year</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manually</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>On application exit</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cacheGroup">
     <property name="title">
      <string>Browser Cache</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="3">
       <widget class="QCheckBox" name="diskCacheCheckBox">
        <property name="text">
         <string>Enable disk cache</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Policy</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QRadioButton" name="cacheKeepButton">
           <property name="toolTip">
            <string>Select to prefer the network</string>
           </property>
           <property name="text">
            <string>Keep cache in sync</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="cachePreferButton">
           <property name="toolTip">
            <string>Select to prefer cached data</string>
           </property>
           <property name="text">
            <string>Use cache whenever possible</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="cacheOfflineButton">
           <property name="toolTip">
            <string>Select to use cached data only</string>
           </property>
           <property name="text">
            <string>Offline browsing mode</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Cache size:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="cacheSizeSpinBox">
        <property name="toolTip">
         <string>Enter the maximum size of the disk cache</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> MB</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>410</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="printGroup">
     <property name="title">
      <string>Printing</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="printBackgroundsCheckBox">
        <property name="toolTip">
         <string>Select to print background colours and images</string>
        </property>
        <property name="text">
         <string>Print background colours and images</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>479</width>
       <height>121</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>singleHelpWindowCheckBox</tabstop>
  <tabstop>saveGeometryCheckBox</tabstop>
  <tabstop>webSuggestionsCheckBox</tabstop>
  <tabstop>startupCombo</tabstop>
  <tabstop>homePageEdit</tabstop>
  <tabstop>setCurrentPageButton</tabstop>
  <tabstop>defaultHomeButton</tabstop>
  <tabstop>defaultSchemeCombo</tabstop>
  <tabstop>javaCheckBox</tabstop>
  <tabstop>javaScriptCheckBox</tabstop>
  <tabstop>jsOpenWindowsCheckBox</tabstop>
  <tabstop>jsClipboardCheckBox</tabstop>
  <tabstop>pluginsCheckBox</tabstop>
  <tabstop>savePasswordsCheckBox</tabstop>
  <tabstop>dnsPrefetchCheckBox</tabstop>
  <tabstop>expireHistory</tabstop>
  <tabstop>diskCacheCheckBox</tabstop>
  <tabstop>cacheKeepButton</tabstop>
  <tabstop>cachePreferButton</tabstop>
  <tabstop>cacheOfflineButton</tabstop>
  <tabstop>cacheSizeSpinBox</tabstop>
  <tabstop>printBackgroundsCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
